### PR TITLE
Rename python classes

### PR DESCRIPTION
**Pull request by arashBehpour**
I renamed for a couple of MCS classes. Just wanted to put my work here so that anyone else can easily finish this ticket.

### CODE REVIEW

**Comment by deanwetherby** [comment]
@arashBehpour Thanks for pushing up your progress on this ticket.

For whomever picks up this ticket:
Let's be sure to rename the files/modules associated with these class fixes. 
One potential issue that I noticed is renaming MCS_Object to object. As you know, object is a python keyword. We will need to think of another noun to replace it.

File: python_api/machine_common_sense/object_.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
-from .mcs_util import MCS_Util
+from .material import Material
+from .util import Util
 
 
-class MCS_Object(object):
+class Object(object):
```
Comment (rartiss55) [inline python_api/machine_common_sense/object_.py:5]: Just some brainstorming if we want to call this something other than object: item, attributes, traits, some combination of those?

Comment (ThomasSchellenbergNextCentury) [reply python_api/machine_common_sense/object_.py:5]: I'm thinking "ObjectMetadata"

**Comment by ThomasSchellenbergNextCentury** [comment]
Closing because I've replaced this PR with https://github.com/NextCenturyCorporation/MCS/pull/179

**Pull request closed without merging**